src/controllers/requests/allUserRequest.js

```diff
@@ -0,0 +1,34 @@
+
+const db = require('../../database/db');
+require('dotenv').config();
+
+
+const getAllRequestByUser = async (request, response) => {
+    try{
+
+        const getQuery = 'SELECT * FROM requests WHERE userid = $1';
+        const userRequests = await db.query(getQuery, [request.user.id]);
+
+        if (userRequests.rows.length === 0){
```

Comment:
You don't need this. The user should get an empty array if no requests has been created.
Its not a user error if they haven't created a request